build/common.sh

```diff
@@ -367,6 +367,8 @@ function kube::build::clean() {
 
   if [[ -d "${LOCAL_OUTPUT_ROOT}" ]]; then
     kube::log::status "Removing _output directory"
+    # this ensures we can clean _output/local/go/cache which is not rw by default
```

Comment:
This is because we `go install` in module mode with GOPATH under `_output`, creating these non-writeable module cache files. We could make `make clean` aware of the go module path but ... that's a bit messy.

I think it generically makes sense to ensure the directory is recursively writeable before trying to remove it.

Reply:
interesting!!